spark/src/main/kotlin/com/adevinta/spark/tokens/Type.kt

```diff
@@ -410,3 +410,6 @@ private fun TextPreview(
         )
     }
 }
+
+public val TextStyle.Highlight: TextStyle
```

Comment:
Just wondering, is this `Highlight` term something defined in MaterialDesign/Spark? Because it has a totally different meaning to me.

Reply:
Yes, here is the link https://zeroheight.com/1186e1705/p/831208-typography/b/45b90c#:~:text=the%20jovial%20queen-,Body%201%20Highlight,-Family

Reply:
however, i think I will go for a lower case to have more consistency with other typo's namings